section3/reflection.md

```diff
@@ -2,16 +2,40 @@
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
 
+    - Workflow felt good to step away and then come back. Looking forward to practicing this through Turing. Normally, I can get lost in my work and bang my head around things.
+
 2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
 
+    - I noticed that sometimes that my estimation of completing things does not always align. There were times where things went quicker than expected and then there were others where I really struggled and wanted to learn more before moving on to the next task.
```

Comment:
Yep!! I think that's ok - to some degree, you'll get better at estimating. And for some things... you still won't get it right. And I think that's ok - that's why a lot of software teams use agile methodologies, so they can quickly shift based on how things are going. Be kind to yourself!!